src/repo.rs

```diff
@@ -8,7 +8,7 @@ use nostr_sdk::prelude::*;
 use nostr_sdk::prelude::{decrypt, encrypt};
 use nostr_sdk::{EventBuilder, Tag};
 use std::time::Duration;
-use url::Url;
+use ::url::Url;
```

Comment:
`nostr_sdk::prelude::*` has exported an ambiguous `﻿url` item, so we need to designate it as root.